websites/N/Naver/presence.ts

```diff
@@ -0,0 +1,496 @@
+const presence = new Presence({
+  clientId: "825307070584586250",
+  injectOnComplete: true
+}),
+  getServiceName = (url = document.location.hostname) => {
+    switch(true){
+      case !!url.match(/tv[.]naver[.]([a-z0-9]+)/):
+        return "NAVER_TV";
+      case !!url.match(/comic[.]naver[.]([a-z0-9]+)/):
+        return "NAVER_WEBTOON";
+      case !!url.match(/now[.]naver[.]([a-z0-9]+)/):
+        return "NAVER_NOW";
+      case !!url.match(/papago[.]naver[.]([a-z0-9]+)/):
+        return "NAVER_PAPAGO";
+      case !!url.match(/blog[.]naver[.]([a-z0-9]+)/):
+        return "NAVER_BLOG";
+      case !!url.match(/([a-z]+)[.]naver[.]([a-z0-9]+)/):
+        return "NAVER";
+      default:
+        break;
+    }
+  },
+  data: {
+    isChecked: boolean,
+    service: string,
+    papagoLangs: Map<string, {
+      code: string,
+      name: string
+    }>,
+    settings?: {
+      id: string,
+      delete?: boolean,
+      data: Array<string>
+    }[],
+    presence: {
+      [key: string]: {
+        env?: boolean,
+        service: "NAVER" | "NAVER_PAPAGO" | "NAVER_NOW" | "NAVER_WEBTOON" | "NAVER_TV" | "NAVER_BLOG" | "ANY",
+        setPresenceData?: () => void,
+        data?: {
+          [key in keyof PresenceData]: {
+            setTo?: unknown
+            if?: {
+              s: {
+                k: unknown,
+                v: unknown,
+                then?: {
+                  v?: unknown,
+                  delete?: boolean
+                },
+                else?: {
+                  v?: unknown,
+                  delete?: boolean
+                }
+              }
+              else?: {
+                k: unknown,
+                v: unknown,
+                then?: {
+                  v?: unknown,
+                  delete?: boolean
+                },
+                else?: {
+                  v?: unknown,
+                  delete?: boolean
+                }
+              }[]
+            }
+          }
+        }
+      }
+    }
+  } = {
+    isChecked: false,
+    service: null,
+    papagoLangs: new Map(),
+    presence: null
+  },
+
+  getLanguage = async (langCode: string) => {
+    if (!langCode) return "Choosing";
+
+    if (langCode === "auto") return "Detecting";
+    if (data.papagoLangs.has(langCode)) return data.papagoLangs.get(langCode).name;
+
+    const baseURL = "https://restcountries.eu/rest/v2/lang",
```

Comment:
Why? You know that you need to explain why you use this api because for now, I'm not accepting this thing.

Reply:
![Screenshot_20210401-111055.png](https://user-images.githubusercontent.com/9348108/113271601-ff60ec80-92da-11eb-8f71-9caa3d575c76.png)

Reply:
Reee I'm not gonna bother explain lmao, fine I will use DOM

Reply:
Idk I just wnated to know lol but if not K then